test/units/plugins/inventory/test_inventory.py

```diff
@@ -32,6 +33,9 @@
 from units.mock.loader import DictDataLoader
 
 
+# ensure ini plugin will read __file__
+os.environ['ANSIBLE_INVENTORY_PLUGIN_INI_EXT'] = 'ini, py,'
```

Comment:
This should be set in individual test cases, possibly using a fixture. Otherwise this will apply to all tests in all files run in the same session.